app/scripts/modules/cloudfoundry/src/loadBalancer/details/loadBalancer.details.html

```diff
@@ -0,0 +1,85 @@
+  <div class="details-panel">
+  <div ng-if="state.loading" class="header">
+    <div class="close-button">
+      <a class="btn btn-link"
+         ui-sref="^">
+        <span class="glyphicon glyphicon-remove" />
+      </a>
+    </div>
+    <div class="horizontal center middle">
+      <loading-spinner size="'small'" />
+    </div>
+  </div>
+
+  <div ng-if="!state.loading" class="header">
+    <div class="close-button">
+      <a class="btn btn-link"
+         ui-sref="^">
+        <span class="glyphicon glyphicon-remove" />
+      </a>
+    </div>
+    <div class="header-text horizontal middle">
+      <i class="fa icon-sitemap"/>
+      <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
+        {{ctrl.loadBalancer.name}}
+      </h3>
+    </div>
+    <div>
+      <div class="actions">
+        <div class="dropdown" uib-dropdown dropdown-append-to-body>
+          <button type="button" class="btn btn-sm btn-primary dropdown-toggle" uib-dropdown-toggle>
+            Load Balancer Actions <span class="caret" />
+          </button>
+          <ul class="dropdown-menu" uib-dropdown-menu role="menu">
+            <li ng-if="ctrl.canDeleteLoadBalancer()">
+              <a href ng-click="ctrl.deleteLoadBalancer()">Delete Load Balancer</a>
+            </li>
+            <li ng-if="!ctrl.canDeleteLoadBalancer()"
+                uib-tooltip="You cannot delete a default service."
```

Comment:
Again, looks like App Engine. Just confirming there's something called a "default service" for CF.